common/compositor/va/varenderer.cpp

```diff
@@ -246,14 +246,13 @@ bool VARenderer::Draw(const MediaState& state, NativeSurface* surface) {
 
   // Get Output Surface.
   OverlayLayer* layer_out = surface->GetLayer();
-  HwcRect<int> layer_out_disp_frame = layer_out->GetDisplayFrame();
-  int xtranslation = layer_out_disp_frame.left;
-  int ytranslation = layer_out_disp_frame.top;
-
+  HwcRect<int> layer_out_display_frame = layer_out->GetDisplayFrame();
```

Comment:
Those lines are not changed, you don't need change?

Reply:
Updated